packages/wonder-stuff-core/src/safe-has-own-property.js

```diff
@@ -0,0 +1,7 @@
+// @flow
+/**
+ * Wrapper to Object.prototype.hasOwnProperty so we only need suppress flow once
+ */
+export const safeHasOwnProperty = (obj: {...}, property: string): boolean =>
```

Comment:
I'll remove this. It's not necessary and I think possibly defeats the purpose by having as an export?